manifests/service.pp

```diff
@@ -44,6 +44,11 @@
   }
 
   if $manage_service {
+    # workaround for redhat's storm-service rpm 
+    file { "/etc/init.d/storm-${name}":
```

Comment:
Could we make it just RedHat specific? I guess limiting to RedHat, Centos version 6 should be enough, right?